index.js

```diff
@@ -177,16 +177,20 @@ function initialize(args) {
     // Do not make modifications to this.
     var originalPathItem = originalApiDoc.paths[openapiPath] || {};
     var pathItem = apiDoc.paths[openapiPath] || {};
-    var pathParameters = Array.isArray(pathModule.parameters) ?
-        [].concat(pathModule.parameters) :
-        [];
+    var pathParameters = pathItem.parameters || [];
+
+    // push all parameters defined in the path module to the path parameter list
+    if (Array.isArray(pathModule.parameters)) {
+      Array.prototype.push.apply(pathParameters, pathModule.parameters);
+    }
+
     pathItem.parameters = pathParameters;
     apiDoc.paths[openapiPath] = pathItem;
 
     Object.keys(pathModule).filter(byMethods).forEach(function(methodName) {
       // operationHandler may be an array or a function.
       var operationHandler = pathModule[methodName];
-      var operationDoc = handleYaml(getMethodDoc(operationHandler));
+      var operationDoc = handleYaml(getMethodDoc(operationHandler)) || pathItem[METHOD_ALIASES[methodName]];
```

Comment:
Shouldn't we allow YAML for both?

Reply:
I'll check this, if we need to take care for yaml on `pathItem` at this point, or if it has already been handled.

Reply:
As far as i see: pathItem is already an object. potential yaml has already been handled.

```js
var originalApiDoc = handleYaml(args.apiDoc);  // #L71 
var apiDoc = copy(originalApiDoc);                     // #L73
var pathItem = apiDoc.paths[openapiPath] || {};  // #179
```
see https://github.com/alrik/express-openapi/blob/feature/api-doc-fallback/index.js#L71 and https://github.com/alrik/express-openapi/blob/feature/api-doc-fallback/index.js#L73 and https://github.com/alrik/express-openapi/blob/feature/api-doc-fallback/index.js#L179